samples/tfm_integration/psa_protected_storage/src/main.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <string.h>
+#include <sys/printk.h>
+#include <psa/storage_common.h>
+#include <psa/protected_storage.h>
+
+#define TEST_STRING_1 "The quick brown fox jumps over the lazy dog"
+#define TEST_STRING_2 "Lorem ipsum dolor sit amet"
+
+void main(void)
+{
+	psa_status_t status = 0;
+
+	printk("Protected Storage sample started. ");
+	printk("PSA Protected Storage API Version %d.%d\n",
+			PSA_PS_API_VERSION_MAJOR, PSA_PS_API_VERSION_MINOR);
+
```

Comment:
Is it worth also displaying the maximum number of PS records, and max record size for the platform in this demo, since those are key pieces of information that are easy to overlook?

Reply:
Yes, I think that would be helpful. However, I wasn't able to find anywhere these values are exposed to the application, appears they're only passed to the TFM partition? If they're not accessible by the application I can add a note to the doc stating that they're printed by the build system

Reply:
Sounds reasonable. After posting this I was wondering the same thing ... can we access those values from the application side? Documenting it in the README seems sufficient, though, since we're at least leaving a signpost for people when they run into issues with record size or count.